### DISCUSSION TITLE
HTML formatting in Long Description

### DISCUSSION THREAD

**Discussion by ppunzi** [PodcastGenerator/PodcastGenerator#670] [Ideas]
I didn't find a thread but I apologize if it has been addressed.  I would like the ability to have html formatting in the long description - right now there is no formatting.  Thanks.

**Comment by charabaruk** [comment]
You can use HTML tags directly in the long description, but yes there's no WYSIWYG editor for the long description field right now. I hope to reintroduce one at some point, but there are other features and fixes which are higher priority for me. Pull requests are always welcome though!

**Reply by ppunzi** [reply]
HTML Tags show up as text when I do it.  I added` </br>` and it shows up as text.  I have also tried `<p>text</p>`.  Maybe it is a version issue?  I am using 3.1 as this is what is available from my provider through softaculous.

**Reply by charabaruk** [reply]
Yeah that's a bug in 3.1 that was fixed in a later version. The entire 3.1.x series of releases are out of support now as well. Could you upgrade to 3.2?

**Reply by ppunzi** [reply]
Unfortunately the Softalicious App doesn't offer the update as it does for other programs. It is 3.1 but if I were to install from scratch it is offering 3.2.6.  I was going to install it manually per the directions but there are so many folders that are new and I can't upload folders - that it would be a major pain to create all the folders and then populate them with files.  I may need to install a new instance and then figure out how to move over the mp3 files and the episode information.

**Reply by charabaruk** [reply]
How do you access the files on your web server normally? If there's FTP, SFTP, or FTPS access, most reasonable clients will automatically create and populate subdirectories when you copy a directory from one machine to another. See if you can access your web server with Filezilla, talk with your hosting company if necessary if they don't normally provide FTP(S) or SFTP access (and look to switch to a better company).

If you _do_ take the route of creating a new PG instance, then what you need to do is copy the contents of the original instance's media/ and images/ directories to their respective directories in the new instance, as well as config.php and the XML files in the original instance's root directory. However I'd expect that things will break due to different file system paths and/or domain names.

**Reply by charabaruk** [reply]
It's very frustrating to me, too, that this app doesn't provide a clean update path when actually performing an upgrade from 3.1.x to 3.2.x is so simple. I hope you're not paying to use Softalicious because if you are, you're getting ripped off.

**Reply by ppunzi** [reply]
I haven't used FTP in a while as I can usually do things through the File System provided by my hosting service.  I was able to do a clean install and move the files and everything works as expected.  Thank you.  Softalicious comes free with the hosting service and is usually good about sending emails about upgrades available to other programs. Hopefully now it will do it for Podcast Generator after this new install.  Thanks for your help.

**Reply by ppunzi** [reply]
Good News.  It appears that now Softalicious knows there is an update from 3.2.6 to 3.2.7 so hopefully that will continue to update.

**Reply by ppunzi** [reply]
Here is what it said to do after the update: 
Please perform following steps to complete the upgrade process :
1. Log in and go to the admin area and select 'Change Podcast Generator Configuration'
2. Save the current configuration (even if you haven't changed anything). This will automatically regenerate the configuration file.
3. From the admin section select Manually regenerate the RSS feed to update your RSS feed to the latest version after the upgrade of Podcast Generation.
4. After performing above steps please provide safe permissions to /home/hortic5/peterzee.rocks/config.php file

My current config.php permission is set to 644 don't think I need to change that.

**Reply by charabaruk** [reply]
None of those steps should be necessary (except perhaps manually regenerating RSS, but not for a patch-level update, e.g. 3.2.6 to 3.2.7). Podcast Generator automatically migrates the configuration file config.php on the first page load after updating the scripts, and won't change the permissions on config.php. This just makes me even more wary of Softalicous.
